### PR TITLE
Bringing in Windows package manifest and pipeline

### PR DESCRIPTION
**Pull request by starnight**
Bring the appxmanifest.xml for APPX/MSIX packaging. Also, add the steps
into the CI pipeline to build the package and upload the artifact.

The first 3 fields of Identity's Version in appxmanifest.xml should be
same as the release version. The fourth field could be build times, or
anything else.

https://phabricator.endlessm.com/T33442

### CODE REVIEW

**Comment by starnight** [comment]
Build failed with:
```
Run cp .buildkite_win/appxmanifest.xml kolibri-windows/
  cp .buildkite_win/appxmanifest.xml kolibri-windows/
  "C:\Program Files (x86)\Windows Kits\10\App Certification Kit\makeappx.exe" pack /d kolibri-windows /p endlesskeyapp.appx
  shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
  env:
    pythonLocation: C:\hostedtoolcache\windows\Python\[3](https://github.com/endlessm/kolibri-app/runs/6284094453?check_suite_focus=true#step:18:3).7.9\x86
ParserError: D:\a\_temp\677d0f0b-9007-[4](https://github.com/endlessm/kolibri-app/runs/6284094453?check_suite_focus=true#step:18:4)9ab-8dcb-be8097da0[6](https://github.com/endlessm/kolibri-app/runs/6284094453?check_suite_focus=true#step:18:6)0b.ps1:3
Line |
   3 |  … x[8](https://github.com/endlessm/kolibri-app/runs/6284094453?check_suite_focus=true#step:18:8)6)\Windows Kits\10\App Certification Kit\makeappx.exe" pack /d koli …
     |                                                             ~~~~
     | Unexpected token 'pack' in expression or statement.

Error: Process completed with exit code 1.
```
Looks like running in PowerShell, instead of command.

**Comment by starnight** [comment]
PowerShell needs the character '**&**' for a command.

**Review by danigm — APPROVED** [review]
Looks good to me

File: .buildkite_win/appxmanifest.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
+  xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10">
+  <Identity Name="EndlessOSFoundation.EndlessKey.app" Version="0.1.10.1" Publisher="CN=Endless OS Foundation LLC, O=Endless OS Foundation LLC, L=San Francisco, S=California, C=US, SERIALNUMBER=7858239, OID.2.5.4.15=Private Organization, OID.1.3.6.1.4.1.311.60.2.1.2=Delaware, OID.1.3.6.1.4.1.311.60.2.1.3=US" ProcessorArchitecture="x64" />
```
Comment (jprvita) [inline .buildkite_win/appxmanifest.xml:7]: `L=` and `S=` should be Wilmington, DE, which is our legal address. We don't have SF address anymore.

Comment (jprvita) [reply .buildkite_win/appxmanifest.xml:7]: Actually, these all need to match the signing certificate. I have addressed it on https://github.com/endlessm/kolibri-app/pull/36.

**Pull request merged**